plugins/finder/jevents/jevents.php

```diff
@@ -384,19 +342,121 @@ protected function getItem($id)
 	 */
 	protected function getStateQuery()
 	{
-		$sql = $this->db->getQuery(true);
+		$db     = $this->db;
+		$query  = $db->getQuery(true);
+
 		// Item ID
-		$sql->select('a.evdet_id as id');
+		$query->select('a.ev_id');
+
```

Comment:
The whole of the finder plugin is built around event details so if we are changing the event state we need to update the entries for all the relevant event detail ids.